### DISCUSSION TITLE
Mistake on license use.

### DISCUSSION THREAD

**Discussion by ballisticfox** [TweakScale/TweakScale#298] [Support] [answered]
Biotronic/TweakScale is licensed under the:
[Creative Commons Attribution-NonCommercial-ShareAlike license](url)

> ShareAlike.
> In addition to the conditions in Section [3(a)](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode#s3a), if You Share Adapted Material You produce, the following conditions also apply.
>     The Adapter’s License You apply must be a Creative Commons license with the same License Elements, this version or later, or a BY-NC-SA Compatible License.
    You must include the text of, or the URI or hyperlink to, the Adapter's License You apply. You may satisfy this condition in any reasonable manner based on the medium, means, and context in which You Share Adapted Material.
    You may not offer or impose any additional or different terms or conditions on, or apply any Effective Technological Measures to, Adapted Material that restrict exercise of the rights granted under the Adapter's License You apply.

This would make the pellinor0/TweakScale also under CC-BY-NC-SA.

This repo was forked from pellinor0/TweakScale and classifies it as a derivative, making it under the CC-BY-NC-SA license.

Thanks.

**Comment by ballisticfox** [comment]
Note, just saw this
https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses/

**Comment by Lisias** [comment, marked as answer]
It was changed to WTFPL by the author. See this [post](https://forum.kerbalspaceprogram.com/index.php?/topic/72554-090-tweakscale-rescale-everything-v150-2014-12-24-1040-utc/&do=findComment&comment=1725543).

Screenshot available [here](https://github.com/TweakScale/TweakScale/raw/master/Docs/LMJ/Biotronic_about_license_001.png).

![](https://github.com/TweakScale/TweakScale/raw/master/Docs/LMJ/Biotronic_about_license_001.png)

And this is the mentioned file on Biotronic's repository:

https://github.com/Biotronic/TweakScale/blob/master/GameData/TweakScale/plugins/Licence.txt

Local mirror: 

https://github.com/TweakScale/TweakScale/blob/upstream/root/master/GameData/TweakScale/plugins/Licence.txt


It's also stated as WTFPL on the [Forum's Thread](https://forum.kerbalspaceprogram.com/index.php?/topic/72554-090-tweakscale-rescale-everything-v150-2014-12-24-1040-utc/).

Local mirror: https://github.com/TweakScale/TweakScale/blob/master/Docs/LMJ/%5B0.90%5D%20TweakScale%20-%20Rescale%20Everything!%20(v1.50%20-%202014-12-24%2010:40%20UTC)%20-%20Add-on%20Releases%20-%20Kerbal%20Sp.01.pdf

**Comment by Lisias** [comment]
@ballisticfox is there's something else you need me to explain to you?

**Comment by ballisticfox** [comment]
Nope thanks for the explanation!
